src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/UserService.java

```diff
@@ -43,6 +43,8 @@ public UserDTO getUser(String userIdentity) throws CTPException {
             .orElseThrow(
                 () -> new CTPException(Fault.BAD_REQUEST, "User not found: " + userIdentity));
 
+    checkIfDeleted(user);
```

Comment:
The same block of code appears numerous times in this class:

    User user =
        userRepository
            .findByIdentity(userIdentity)
            .orElseThrow(() -> new CTPException(Fault.BAD_REQUEST, "User not found"));

    checkIfDeleted(user);

Prime candidate for abstraction.

DRY